services/dkp-insights/0.5.0-beta.1/helmrelease.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
```

Comment:
my only other suggestion might be to move this HelmRelease behind a Flux Kustomization like the insights management app does: 

https://github.com/mesosphere/kommander-applications/blob/main/services/dkp-insights-management/0.5.0/helmrelease.yaml

this can make things more flexible if we need to add jobs to automate upgrade/install issues in the future (you can check out the [Loki app](https://github.com/mesosphere/kommander-applications/tree/main/services/grafana-loki/0.69.16) as an example). If we don't do this now, moving the HelmRelease later on would likely cause things to be uninstalled/reinstalled. 

I'm also good keeping this as-is since insights is probably less likely to need some kind of automation like that!